CONTRIBUTING.md

```diff
@@ -0,0 +1,379 @@
+# Contributing to `aroma`
+
+Welcome to the `aroma` repository! We're excited you're here and want to contribute.
+
+These guidelines are designed to make it as easy as possible to get involved.
+If you have any questions that aren't discussed below, please let us know by opening an [issue][link_issues]!
+
+Before you start you'll need to set up a free [GitHub][link_github] account and sign in.
+Here are some [instructions][link_signupinstructions].
+
+Already know what you're looking for in this guide? Jump to the following sections:
+
+- [Contributing to `aroma`](#contributing-to-aroma)
+  - [Joining the conversation](#joining-the-conversation)
+  - [Contributing small documentation changes](#contributing-small-documentation-changes)
+  - [Contributing through GitHub](#contributing-through-github)
+  - [Understanding issues, milestones and project boards](#understanding-issues-milestones-and-project-boards)
+    - [Issue labels](#issue-labels)
+  - [Making a change](#making-a-change)
+    - [1. Comment on an existing issue or open a new issue referencing your addition](#1-comment-on-an-existing-issue-or-open-a-new-issue-referencing-your-addition)
+    - [2. Fork to your GitHub profile](#2-fork-the-aroma-repositorylink_aroma-to-your-github-profile)
+    - [3. Run the developer setup](#3-run-the-developer-setup)
+    - [4. Make the changes you've discussed](#4-make-the-changes-youve-discussed)
+    - [5. Test your changes](#5-test-your-changes)
+      - [Changes to documentation](#changes-to-documentation)
+    - [6. Submit a pull request](#6-submit-a-pull-request)
+    - [Pull Requests](#pull-requests)
+    - [Pull Request Checklist (For Fastest Review):](#pull-request-checklist-for-fastest-review)
+    - [Comprehensive Developer Guide](#comprehensive-developer-guide)
+  - [Style Guide](#style-guide)
+    - [Writing in ReStructuredText](#writing-in-restructuredtext)
+  - [Recognizing contributors](#recognizing-contributors)
+  - [Thank you!](#thank-you)
+
+Don't know where to get started?
+Read [Joining the conversation](#joining-the-conversation) and pop into
+Gitter to introduce yourself! Let us know what your interests are and we
+will help you find an issue to contribute to. Thanks so much!
+
+## Joining the conversation
+
+`aroma` is a young project maintained by a growing group of enthusiastic developers&mdash; and we're excited to have you join!
+Most of our discussions will take place on open [issues][link_issues].
+We also maintain a [gitter chat room][link_gitter] for more informal conversations and general project updates.
+
+There is significant cross-talk between these two spaces, and we look forward to hearing from you in either venue!
+As a reminder, we expect all contributions to `aroma` to adhere to our [code of conduct][link_coc].
```

Comment:
I don't think we're using Gitter, so I'd just mention the issues.

Reply:
I forgot to make that a question to everyone involved in this project.